### DISCUSSION TITLE
Moonlight setting 'Mute host PC speakers while streaming' doesn't work with Sunshine

### DISCUSSION THREAD

**Discussion by CubicPants** [LizardByte/community#155] [Sunshine] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Explanation is in the title.
The setting in Moonlight doesn't seem to have any effect, but it does work with NVIDIA Gamestream.

### Expected Behavior

Audio should be muted on the host PC while streaming.

### Host Operating System

Windows

### Operating System Version

Windows 11 Pro Version 10.0.22621 Build 22621

### Architecture

64 bit

### Sunshine commit or version

0.18

### Package

Windows - installer

### GPU Type

Nvidia

### GPU Model

GeForce RTX 2070

### GPU Driver/Mesa Version

527.57

**Comment by mlbrnm** [comment]
Do you have some sort of virtual sink audio driver installed? You can use the Steam ones found here:

https://github.com/FCGWolfe/SteamLinkAudioDrivers

I had to install these as well to get audio out of Moonlight and mute the host PC. It should maybe be part of the docs/FAQ as I've seen it mentioned a few times if this is indeed your issue.

**Comment by AlexKuai** [comment]
I also have the same question. My audio settings in Sunshine are the default settings during installation. Do you need to modify them?

**Comment by CubicPants** [comment]
> Do you have some sort of virtual sink audio driver installed? You can use the Steam ones found here:
> 
> https://github.com/FCGWolfe/SteamLinkAudioDrivers
> 
> I had to install these as well to get audio out of Moonlight and mute the host PC. It should maybe be part of the docs/FAQ as I've seen it mentioned a few times if this is indeed your issue.

I've tried the following configs in Sunshine, but same issue. Audio plays just fine in Moonlight, but the host PC isn't muted.

Host audio devices:
```
===== Device =====
Device ID          : {0.0.0.00000000}.{4091eefb-3d55-48fe-b478-c7715ed048d4}
Device name        : Desktop (Realtek(R) Audio)
Adapter name       : Realtek(R) Audio
Device description : Desktop
Device state       : Active

===== Device =====
Device ID          : {0.0.0.00000000}.{cbd89264-5af0-4077-ba81-6ccd6ee5f102}
Device name        : Speakers (Steam Streaming Speakers)
Adapter name       : Steam Streaming Speakers
Device description : Speakers
Device state       : Active
```

Sunshine Settings:
```
Audio Sink {0.0.0.00000000}.{4091eefb-3d55-48fe-b478-c7715ed048d4}
Virtual Sink {0.0.0.00000000}.{cbd89264-5af0-4077-ba81-6ccd6ee5f102}
```

**Comment by ReenigneArcher** [comment, marked as answer]
Just set the virtual sink only, not both. You can use virtual audio cable -> https://vb-audio.com/Cable/
